<ui version="4.0" >
 <class>SoundLibraryExportDialog_UI</class>
 <widget class="QDialog" name="SoundLibraryExportDialog_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="browseBtn" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>10</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="drumkitPathTxt" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="drumkitList" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>50</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Export</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
